source/basic.tex

```diff
@@ -3214,7 +3214,7 @@
 A \defn{potentially-overlapping subobject} is either:
 \begin{itemize}
 \item a base class subobject, or
-\item a non-static data member
+\item a non-static data member that is of an object type and
```

Comment:
This change seems at odds with [dcl.attr.nouniqueaddr] p1, which says

> The attribute-token no_unique_address specifies that a non-static data member is a potentially-overlapping
subobject (6.7.2).

Reply:
I've made corresponding change in [dcl.attr.nouniqueaddr] p1 and added a note for reference type.